Charts/Classes/Renderers/ChartYAxisRenderer.swift

```diff
@@ -70,6 +70,10 @@ public class ChartYAxisRenderer: ChartAxisRendererBase
         
         let rawInterval = range / Double(labelCount)
         var interval = ChartUtils.roundToNextSignificant(number: Double(rawInterval))
+        if(_yAxis.valueFormatter?.maximumFractionDigits == 0)
```

Comment:
Yes please separate subjects to different PRs. Every subject should be on a different branch - and then creating a PR from that branch will create a separate PR.